src/papilio/vendors/keyboardio/model01.h

```diff
@@ -0,0 +1,32 @@
+/* -*- mode: c++ -*-
```

Comment:
This seems to be pretty much the same as `src/kaleidoscope_simulator/vendors/keyboardio/model01.h`, sans the namespace.